lms/envs/test.py

```diff
@@ -75,6 +75,9 @@
 
 FEATURES['ENABLE_COMBINED_LOGIN_REGISTRATION'] = True
 
+# Toggles Open Badging for testing
```

Comment:
This comment doesn't add anything useful:
- We're changing the value, so it's clear it's being toggled.
- The reason this file exists is to override values for testing,
  so we don't need to call that out again.

Let's remove it.